### PR TITLE
Release notes 344

### PR DESCRIPTION
**Pull request by kaitlynmortimer**

### CODE REVIEW

**Comment by kaitlynmortimer** [comment]
Please note that I've included the ticket numbers in the "Fixes" section for reference. I'll remove them once the list has been finalized.

File: content/release/v3/3.4/ReleaseNotes344.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+title: "Contrast 3.4.4 - September 2017"
+description: "Contrast 3.4.4 September 2017"
+tags: "3.4.4 September Release Notes"
+-->
+
+Congratulations, Ruby! Version 1.0.0 has arrived along with new bugtracker integration features like configuration for customized URLs and automatic ticket creation. 
+
+## Fixes
+
+* Added event tag and source type options CONTRAST-17145, CONTRAST-17322
```
Comment (SteveFeldman) [inline content/release/v3/3.4/ReleaseNotes344.md]: Thought we were going to stop referencing JIRA tickets in the notes. Is that not the case? If it is, let's remove all of the ticket numbers.

File: content/release/v3/3.4/ReleaseNotes344.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+* Have a customized VSTS or TFS URL? Go ahead and set up your bugtracker in the **Integrations** page.
+
+## Agent Updates
+
+* **Java Agent Summary:** We added protection for CVE-2017-9804 as well as requirements to [level 2 rules](installation-javapolicy.html#level2) to support improved summary and detail view of traces on Contrast. We also improved the accuracy of an XXE rule, and fixed a bug in which the agent reported an incorrect library count when you used `contrast.override.appname`.
+
+* **.NET Agent Summary:** We improved agent performance and accuracy, and added support for analysis of SignalR-based applications. We also fixed a bug where certain special characters could prevent the agent from sending vulnerability details to Contrast.
+
+* **Node.js Agent Summary:** While we were getting ready to support node 8, we did a ton of housekeeping. Among our repairs are issues with our method for bundling dependencies, using lazy routers in Express and Sylog client usage on Windows.
```
Comment (SteveFeldman) [inline content/release/v3/3.4/ReleaseNotes344.md]: You might want to discuss Hapi support with @cixel

File: content/release/v3/3.4/ReleaseNotes344.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* Got rid of instrumentation validation errors for .NET servers CONTRAST-16959
+* Mended menu options for Node applications CONTRAST-16548
+* Restored the "Never" option for Password and Inactive Account Expiration CONTRAST-16885
+* Test Diagnostics connection after disabling Proxy CONTRAST-16841
+* Re-populating SAML translate CONTRAST-16619
+
+
+## Improvements 
+
+* Our response to Struts2 vulnerabilities: An update to Contrast on September 9 that adds CVE 2017-9805 to our library inventory data as well as an update to our Protect deserialization rule to stop attackers from exploiting this particular vulnerability.
```
Comment (dhafley) [inline content/release/v3/3.4/ReleaseNotes344.md]: Does this apply to EOP?

File: content/release/v3/3.4/ReleaseNotes344.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+* Contrast now supports user authentication using Active Directory Federation Services! To set up the feature, download the *pem* file from your SAML provider, and upload it your **Single Sign-On** page in Contrast.
```
Comment (dhafley) [inline content/release/v3/3.4/ReleaseNotes344.md]: .....from your Identity Provider......

File: content/release/v3/3.4/ReleaseNotes344.md (hunk 3 above)
Comment (kaitlynmortimer) [reply content/release/v3/3.4/ReleaseNotes344.md]: Yes - just confirmed with Kaushik.

**Pull request merged**